pkg/api/norman/customization/aks/listers.go

```diff
@@ -120,6 +123,92 @@ func (s sortableVersion) Less(a, b int) bool {
 	return version.Compare(s[a], s[b], "<")
 }
 
+type KubernetesUpgradeVersion struct {
+	Version string `json:"version"`
+	Enabled bool   `json:"enabled"`
+}
+
+type KubernetesUpgradeVersions []*KubernetesUpgradeVersion
+
+func (s KubernetesUpgradeVersions) Len() int {
```

Comment:
This shouldn't be needed, it's already a slice

Reply:
Are you sure? https://play.golang.org/p/DJJNS35CGzr

Also, it's done for the very much the same situation: https://github.com/rancher/rancher/blob/ef760ce7606af9ef589cc1dc89253acc27609b5b/pkg/api/norman/customization/aks/listers.go#L111

Reply:
Unless `sort` doesn't use `len` for slices already. My bad probably. :)

Reply:
We are implementing this interface https://pkg.go.dev/sort#Interface